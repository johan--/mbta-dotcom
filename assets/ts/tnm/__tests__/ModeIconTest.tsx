import React from "react";
import renderer from "react-test-renderer";
import ModeIcon from "../components/ModeIcon";

it("renders a small icon if supplied a type", () => {
  const tree = renderer.create(<ModeIcon type="green_line" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a station icon if supplied a type that doesn't have an imported icon", () => {
  const tree = renderer.create(<ModeIcon type="unknown" />).toJSON();
  expect(tree).toMatchSnapshot();
});
