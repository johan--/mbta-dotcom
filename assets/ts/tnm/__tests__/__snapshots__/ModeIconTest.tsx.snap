// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a small icon if supplied a type 1`] = `
<span
  className="m-tnm-sidebar__mode-icon"
  dangerouslySetInnerHTML={
    Object {
      "__html": "SVG",
    }
  }
/>
`;

exports[`renders a station icon if supplied a type that doesn't have an imported icon 1`] = `
<span
  className="m-tnm-sidebar__mode-icon"
  dangerouslySetInnerHTML={
    Object {
      "__html": "SVG",
    }
  }
/>
`;
